test/stage1/behavior/bitreverse.zig

```diff
@@ -2,80 +2,68 @@ const std = @import("std");
 
     // using comptime_ints, signed, positive
-    expect(@bitreverse(i0, 0) == 0);
-    expect(@bitreverse(i8, @bitCast(i8, u8(0x92))) == @bitCast(i8, u8(0x49)));
-    expect(@bitreverse(i16, @bitCast(i16, u16(0x1234))) == @bitCast(i16, u16(0x2c48)));
-    expect(@bitreverse(i24, @bitCast(i24, u24(0x123456))) == @bitCast(i24, u24(0x6a2c48)));
-    expect(@bitreverse(i32, @bitCast(i32, u32(0x12345678))) == @bitCast(i32, u32(0x1e6a2c48)));
-    expect(@bitreverse(i40, @bitCast(i40, u40(0x123456789a))) == @bitCast(i40, u40(0x591e6a2c48)));
-    expect(@bitreverse(i48, @bitCast(i48, u48(0x123456789abc))) == @bitCast(i48, u48(0x3d591e6a2c48)));
-    expect(@bitreverse(i56, @bitCast(i56, u56(0x123456789abcde))) == @bitCast(i56, u56(0x7b3d591e6a2c48)));
-    expect(@bitreverse(i64, @bitCast(i64, u64(0x123456789abcdef1))) == @bitCast(i64, u64(0x8f7b3d591e6a2c48)));
-    expect(@bitreverse(i128, @bitCast(i128, u128(0x123456789abcdef11121314151617181))) == @bitCast(i128, u128(0x818e868a828c84888f7b3d591e6a2c48)));
+    expect(@bitReverse(u8, u8(0)) == 0);
+    expect(@bitReverse(i8, @bitCast(i8, u8(0x92))) == @bitCast(i8, u8(0x49)));
+    expect(@bitReverse(i16, @bitCast(i16, u16(0x1234))) == @bitCast(i16, u16(0x2c48)));
+    expect(@bitReverse(i24, @bitCast(i24, u24(0x123456))) == @bitCast(i24, u24(0x6a2c48)));
+    expect(@bitReverse(i32, @bitCast(i32, u32(0x12345678))) == @bitCast(i32, u32(0x1e6a2c48)));
+    expect(@bitReverse(i40, @bitCast(i40, u40(0x123456789a))) == @bitCast(i40, u40(0x591e6a2c48)));
+    expect(@bitReverse(i48, @bitCast(i48, u48(0x123456789abc))) == @bitCast(i48, u48(0x3d591e6a2c48)));
+    expect(@bitReverse(i56, @bitCast(i56, u56(0x123456789abcde))) == @bitCast(i56, u56(0x7b3d591e6a2c48)));
+    expect(@bitReverse(i64, @bitCast(i64, u64(0x123456789abcdef1))) == @bitCast(i64, u64(0x8f7b3d591e6a2c48)));
+    expect(@bitReverse(i128, @bitCast(i128, u128(0x123456789abcdef11121314151617181))) == @bitCast(i128, u128(0x818e868a828c84888f7b3d591e6a2c48)));
 
-    // using comptime_ints, signed, negative. Compare to runtime ints returned from llvm.
-    var neg5: i5 = minInt(i5) + 1;
-    expect(@bitreverse(i5, minInt(i5) + 1) == @bitreverse(i5, neg5));
+    // using signed, negative. Compare to runtime ints returned from llvm.
     var neg8: i8 = -18;
-    expect(@bitreverse(i8, -18) == @bitreverse(i8, neg8));
+    expect(@bitReverse(i8, i8(-18)) == @bitReverse(i8, neg8));
     var neg16: i16 = -32694;
-    expect(@bitreverse(i16, -32694) == @bitreverse(i16, neg16));
+    expect(@bitReverse(i16, i16(-32694)) == @bitReverse(i16, neg16));
     var neg24: i24 = -6773785;
-    expect(@bitreverse(i24, -6773785) == @bitreverse(i24, neg24));
+    expect(@bitReverse(i24, i24(-6773785)) == @bitReverse(i24, neg24));
     var neg32: i32 = -16773785;
-    expect(@bitreverse(i32, -16773785) == @bitreverse(i32, neg32));
-    var neg40: i40 = minInt(i40) + 12345;
-    expect(@bitreverse(i40, minInt(i40) + 12345) == @bitreverse(i40, neg40));
```

Comment:
Why remove these?

Reply:
because they are tautologies. They are not testing bitreverse.

Reply:
At the time that code was written, I'm not sure about now, the comment at the top of the block explained correctly what was happening. The LHS values were calculated by comptime code in the Zig compiler while the RHS values are calculated on the LLVM side. These tests were used to make sure the two match.

Reply:
@vegecode is correct, and it's still true. `@bitreverse(i32, -16773785)` exercises comptime evaluation; `@bitreverse(i32, neg32)` with `neg32` as a `var` exercises runtime evaluation.